### PR TITLE
SWMAAS-1454 routing step card adjustments

### PR DESCRIPTION
**Pull request by npike**
This PR updates the routing cards to match the design.

**Notes**
- Updated Off Route Scenario to extend RoutingActivity, much like Walk Time Scenario, and Voice Prompt Scenario.  Needed to do this because I removed the navigation_overlay.xml include that was being used its own layout.
- Needed to update the custom NavigationOverlayView to handle having a click listener assigned to it.  ViewPagers (which this custom class extends) don't normally utilize OnClickListeners.
- made `mapManager` in RouteActivity public so that OffRouting Scenario could use it.

_Demo: Before and After_
Screenshot_1571955410_Screenshot_1572040586

_Demo: swiping through maneuvers_
swmaas1454

### CODE REVIEW

File: Samples/kotlin/src/main/res/drawable/ic_expand_more_24px.xml [hunk 1]
```diff
@@ -4,6 +4,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#ff8e8e93"
```
Comment (npike) [inline Samples/kotlin/src/main/res/drawable/ic_expand_more_24px.xml]: This shouldn't be black.  Should be this gray color according to design instead.

File: Samples/kotlin/src/main/res/layout/activity_routing.xml [hunk 2]
```diff
@@ -16,9 +16,15 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <include
-        android:id="@+id/nav_overlay_container"
-        layout="@layout/navigation_overlay" />
+    <com.phunware.kotlin.sample.NavigationOverlayView
```
Comment (npike) [inline Samples/kotlin/src/main/res/layout/activity_routing.xml]: No need to use an include for this, as there's only a single activity layout now.  We can just use the custom View directly in the layout.

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt [hunk 3]
```diff
@@ -89,13 +90,28 @@ class NavigationOverlayView @JvmOverloads constructor(context: Context, attrs: A
     private lateinit var navigator: Navigator
     private lateinit var adapter: ManeuverPagerAdapter
 
+    /**
+     * Since this custom view extends a ViewPager, we need to handle "click listening" on our own.
+     */
+    private var clickListener: OnClickListener? = null
+
+    init {
```
Comment (dtran-phunware) [inline Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt]: loving the peek look

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt [hunk 4]
```diff
@@ -89,13 +90,28 @@ class NavigationOverlayView @JvmOverloads constructor(context: Context, attrs: A
...
+    }
+
     private val pageChangeListener = object : ViewPager.SimpleOnPageChangeListener() {
         override fun onPageSelected(position: Int) {
             val pair = adapter.getItem(position)
             navigator.setCurrentManeuver(pair.mainPos)
         }
     }
 
+    override fun setOnClickListener(l: OnClickListener?) {
```
Comment (dtran-phunware) [inline Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt]: nitpicky, but do we want a more descriptive parameter name than "l" like listener?

File: Samples/kotlin/src/main/res/layout/activity_routing.xml (hunk 2 above)
Comment (dtran-phunware) [reply Samples/kotlin/src/main/res/layout/activity_routing.xml]: hope i dont get merge conflict, but shouldnt be too difficult to resolve

Comment (npike) [reply Samples/kotlin/src/main/res/layout/activity_routing.xml]: Probably merge conflicts will happen, just need to replace the old include with this block.

File: Samples/kotlin/src/main/res/layout/item_maneuver.xml [hunk 5]
```diff
@@ -1,31 +1,65 @@
...
+
+    <!-- FrameLayout is used to wrap CardView to help facilitate showing multiple cards -->
+    <!-- in the maneuver viewpager.  (otherwise the cards would be edge to edge) -->
+
+    <android.support.v7.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:elevation="12dp"
+        android:innerRadius="0dp"
+        android:shape="ring"
```
Comment (dtran-phunware) [inline Samples/kotlin/src/main/res/layout/item_maneuver.xml]: 😮

**Comment by npike** [comment]
Argh, looks like `OneWayRoutingActivity` should also be updated to extend RoutingActivity, since its using the old navigation overlay!

File: Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt (hunk 4 above)
Comment (dtran-phunware) [reply Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt]: looks good

**Pull request merged**